source/configuration.txt

```diff
@@ -65,11 +65,12 @@ configured via the System Property. See :ref:`cache-configuration`.
 Input Configuration
 --------------------
 
-The following options for reading from MongoDB are available:
+The following properties for reading from MongoDB are available:
 
 .. note::
-   If setting these connector input configurations via ``SparkConf``,
-   prefix these settings with ``spark.mongodb.input.``.
+
+   If you are using ``SparkConf``, prefix ``spark.mongodb.input.`` to
+   each property.
```

Comment:
Suggestion: I like what you've done to streamline, but I think we've lost a little bit of context. Perhaps:
```suggestion
   If you use ``SparkConf`` to set the connector's input configurations, 
   prefix ``spark.mongodb.input.`` to each property.
```